prime-router/settings/organizations.yml

```diff
@@ -545,6 +545,69 @@
           initialTime: 00:00
```

Comment:
I have on my todo list to get rid of either organizations.yml or organizations-local.yml.   I see no reason to have both.    Unfortunately, its a ways down on that list....